### PR TITLE
add learn more landing page

### PR DESCRIPTION
**Pull request by mcbobby7**
Add learn more landing page

### CODE REVIEW

File: src/components/learnMore/components/LearnMore.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
```
Comment (EngrEric) [inline src/components/learnMore/components/LearnMore.js]: remove the space here

File: src/components/learnMore/components/LearnMore.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { Icon } from 'antd';
+
+import { LEARN_MORE_CLASSNAMES, LERAN_MORE_DATA } from '../constants';
+
+const { HEADER, IMAGE, TEXT, BODY, ICON } = LEARN_MORE_CLASSNAMES;
+
+const learn_more = LERAN_MORE_DATA.map(({ key, image, header, body }) => (
```
Comment (EngrEric) [inline src/components/learnMore/components/LearnMore.js]:
```suggestion
const learnMore
```
we use camelcase.

File: src/components/learnMore/components/LearnMore.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </div> 
+)); 
+
+class LearnMore extends React.Component {
+    render() {
+        return (
+            <div>
+                <div className={HEADER}>
+                </div>
+                {learn_more}
```
Comment (EngrEric) [inline src/components/learnMore/components/LearnMore.js]: change to camelcase

File: src/components/learnMore/components/styles.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    -webkit-transform: scale(1.6);
+    -ms-transform: scale(1.6);
+    transform: scale(1.6);
+}
+
+.text {
+    width: 900px;
+    height: 100%;
+    font-size: 90%;
+    text-align: center;
```
Comment (EngrEric) [inline src/components/learnMore/components/styles.css:41]: You may try to` justify` the text

**Comment by EngrEric** [comment]
@mcbobby7  this looks good. just little changes and it can be merged down.
also remember to bump version

File: src/components/learnMore/components/styles.css (hunk 4 above)
Comment (mcbobby7) [reply src/components/learnMore/components/styles.css:41]: @engreric sorry I don't understand

Comment (EngrEric) [reply src/components/learnMore/components/styles.css:41]: Try setting the text align to  `justify`and see if it looks better

File: src/components/app/components/banner.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
 import React, { Component } from "react";
+import { Link } from 'react-router-dom';
```
Comment (Simpcyclassy) [inline src/components/app/components/banner.js]: I think there's no need to import `Link`,  there's one for the app already in `src/components/app/components/Applink`. For a better understanding you may checkout `Navbar.js` and `SideMenu.js`

**Comment by Simpcyclassy** [comment]
Again, in naming your branch, you do not need to add `origin`, the `origin` thing which I think you saw on your editor is telling you the branch is from a remote source.

According to the [engineering conventions](https://github.com/Enye-Cohort-1/Engineering-Conventions) which you will like to go through the method is;
*create a feature/bug branch with the following pattern `feature/[your name]/[name of feature]`

Every other thing looks good to me to me when you make changes.

**Comment by EngrEric** [comment]
you have conflicts @mcbobby7

File: bash.exe.stackdump [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Stack trace:
```
Comment (EngrEric) [inline bash.exe.stackdump]: what is file doing?

File: sh.exe.stackdump [hunk 7]
```diff
@@ -0,0 +1,2 @@
+Stack trace:
```
Comment (EngrEric) [inline sh.exe.stackdump]: this one too

File: src/components/learnMore/components/LearnMore.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Icon } from 'antd';
+import { LEARN_MORE_CLASSNAMES, LERAN_MORE_DATA } from '../constants';
+
+const { HEADER, IMAGE, TEXT, BODY, ICON } = LEARN_MORE_CLASSNAMES;
+
+const learn_more = LERAN_MORE_DATA.map(({ key, image, header, body }) => (
```
Comment (EngrEric) [inline src/components/learnMore/components/LearnMore.js]: change this to `learn_more` to camel-case

**Pull request closed without merging**